src/settings/POLinesLimit.js

```diff
@@ -117,7 +120,7 @@ class POLinesLimit extends Component {
                   component={TextField}
                   label={<FormattedMessage id="ui-orders.settings.setPOLInesLimit" />}
                   name="polines_limit"
-                  placeholder="999"
+                  placeholder={get(poLineLimit, 'value', '999')}
```

Comment:
As I can see this code isn't related to the ticket, maybe it make sense to push it within a separate story/ticket. 
But even in that case the logic doesn't look good to me - if value is entered it's better to see it as an input value. not a placeholder